src/cell.css

```
@@ -0,0 +1,15 @@
+$size: 100px;
+
+$start-days:
+  "mon" 0,
+  "tue" 102px,
+  "wed" 204px,
+  "thu" 306px,
+  "fri" 408px, 
+  "sat" 510px,
+  "sun" 612px;
```

```suggestion
$start-days:
  "mon" 1,
  "tue" 2,
  "wed" 3,
  "thu" 4,
  "fri" 5, 
  "sat" 6,
  "sun" 7;
```
And calculate values in the loop